Dockerfile

```
@@ -50,6 +50,18 @@ FROM core as development
 # Copy all sources, not only runtime-required files
 COPY . /app/
 
+# Only the M1 Mac images need these packages installed
+ARG TARGETPLATFORM
+RUN if [ "$TARGETPLATFORM" = "linux/arm64" ]; \
+    then apt-get update && \
+        apt-get install -y \
+        build-essential \
+        gcc \
+        python-dev && \
```

We usually indent packages to distinguish them from commands:

```suggestion
          build-essential \
          gcc \
          python-dev && \
```